neuhatch/views.py

```diff
@@ -131,8 +131,14 @@ def search():
         max_results -- Integer, the most tweets to return
     """
     query = request.args.get('q')
+
+    if request.args.get('max_results') is None:
```

Comment:
Use the ```default``` keyword argument to ```args.get```: http://werkzeug.pocoo.org/docs/0.10/datastructures/#werkzeug.datastructures.MultiDict.get